packages/react-components/components/BackButton.tsx

```diff
@@ -27,7 +27,7 @@ const SIZE = 16
 
 const style = StyleSheet.create({
   container: {
-    padding: 5,
+    marginLeft: -2,
```

Comment:
I'm weary of changing this. Have you confirmed it looks right in all the places it's used? On iOS and android?